pkg/config/validator.go

```diff
@@ -117,6 +120,8 @@ func registerBindingsValidator(validate *validator.Validate, trans ut.Translator
 		conflictingPluginRepoTag:    "{0}{1}",
 		conflictingPluginVersionTag: "{0}{1}",
 		invalidPluginDefinitionTag:  "{0}{1}",
+		invalidPluginRBACTag:        "RBAC of {0} and {1} are different. But have to be identical.",
+		invalidPluginDefaultNSTag:   "Default namespace of {0} and {1} are different. But have to be identical.",
```

Comment:
I think it would be great to provide info about the specific plugin key - we already have such data so it will be easy.
Then the messages could be something like:
```suggestion
		invalidPluginRBACTag:        "RBAC configuration for plugin '{0}' used in  '{0}' and '{1}' bindings must be identical when used together",
		invalidPluginDefaultNSTag:   "Default namespace for plugin '{0}' used in '{0}' and '{1}' bindings must be identical when used together.",
```

BTW I think later after impl we can remove the verification check for default namespaces. Merging `context.kubeconfig` will be relatively eas - we can take the last value 🤔  But let's keep it for now.